.goreleaser.yaml

```diff
@@ -56,10 +56,26 @@ homebrew_casks:
     hooks:
       post:
         install: |
+          # remove quarantine xattr (note we don't do anything with signatures yet)
           if system_command("/usr/bin/xattr", args: ["-h"]).exit_status == 0
             system_command "/usr/bin/xattr", args: ["-dr", "com.apple.quarantine", "#{staged_path}/cu"]
           end
 
+          # Install shell completions
+          bash_completion_dir = "#{prefix}/etc/bash_completion.d"
```

Comment:
Looks like `goreleaser` as built-in completion thingies?

```yaml
    completions:
      bash: "completions/goreleaser.bash"
      zsh: "completions/goreleaser.zsh"
      fish: "completions/goreleaser.fish"
```
https://github.com/goreleaser/goreleaser/blob/main/.goreleaser.yaml#L234-L237

Generated by a hook here https://github.com/goreleaser/goreleaser/blob/main/.goreleaser.yaml#L11

Reply:
oh nice, i saw these but missed the hook technique which uses `go run` to generate them

https://github.com/goreleaser/goreleaser/blob/main/scripts/completions.sh